ecs-cli/modules/cli/local/docker.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2015-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package local
+
+import (
+	"os"
+
+	"github.com/docker/docker/client"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// minDockerAPIVersion is the oldest Docker API version supporting the operations used by "local" sub-commands.
+	minDockerAPIVersion = "1.27"
```

Comment:
How'd you pick this version?

Reply:
I picked the same version that the endpoints container requires. The operations that we use so far in "ecs local" are all supported in [v1.27](https://docs.docker.com/engine/api/v1.27/).

I can say that the endpoints container uses 1.27 and link to the code ?

Reply:
Sure 👍🏻

Reply:
Done!

Reply:
Actually I'd prefer it if the comment also said that all the operations in ecs local are supported in v1.27

Its hypothetically possible that the "ecs local" commands would need a newer Docker client version than the endpoints container.

The current comment might lead future maintainers to think that the commands and the endpoints container have to use the same Docker Client version. And that's not true.

Reply:
👍 makes sense, done!